scripts/check_coverage.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
any particular reason why there is 2 script files instead of one? Seems like both are simple enough that it could be covered with a single one (pun intended).

Reply:
I had them split mainly for organizational reasons; so that each script performed a specific task.  

I've tested merging them together into one single script and got it working (passing, failing and rendering the detailed html report file); so I could implement this change if desired.  

Please let me know.

Reply:
we can merge as is. Thanks